app/controllers/publish/providers/schools_controller.rb

```diff
@@ -3,59 +3,69 @@
 module Publish
   module Providers
     class SchoolsController < PublishController
-      def index
-        authorize provider, :can_list_sites?
+      before_action :pundit
```

Comment:
Not sure if this is correct. We had specific pundit policies for some of these actions but this is removing them all for the same policy eg `authorize provider, :show?`

Reply:
I might be reading it wrong, but it looks like all these methods are just :show? aliases ?

Reply:
```
alias can_list_sites? show?
  alias can_create_sites? show?
  alias can_create_course? show?
  alias edit? show?
  alias update? show?
  alias destroy? show?
  alias build_new? show?
  alias can_list_training_providers? show?
  alias index? new?
  alias create? show?
  alias delete? show?
```